backend/app/api/endpoints/health.py

```
@@ -0,0 +1,70 @@
+
+from fastapi import APIRouter, Depends
+from sqlalchemy import text
+from sqlalchemy.orm import Session
+
+from app.api.dependencies import get_db
+
+router = APIRouter()
+
+
+@router.get("/health")
+def health_check(db: Session = Depends(get_db)):
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)